urllib3/response.py

```diff
@@ -327,9 +389,19 @@ def read(self, amt=None, decode_content=None, cache_content=False):
                     # no harm in redundantly calling close.
                     self._fp.close()
                     flush_decoder = True
+                    if (self.strict_content_length and
+                       self.length_remaining not in (0, None)):
```

Comment:
This doesn't look to follow the style of the rest of the project and should be indented more. I suspect you're also following PEP8 with a line length of 79. That is not what this project follows.